Sources/XCTest/Public/XCAbstractTest.swift

```diff
@@ -36,20 +36,43 @@ open class XCTest {
     /// testRunClass. If the test has not yet been run, this will be nil.
     open private(set) var testRun: XCTestRun? = nil
 
+    internal var performTask: Task<Void, Never>?
```

Comment:
This is only used in `USE_SWIFT_CONCURRENCY_WAITER` mode, right? Can it move inside the `#if` block?

Reply:
Ah, yes, this should be inside #if block